project/ParadoxSignatureDirective.scala

```diff
@@ -0,0 +1,55 @@
+package akka
+
+import sbt._
+import java.io.{File, FileNotFoundException}
+
+import com.lightbend.paradox._
+import markdown.Writer
```

Comment:
Looks like this is imported by `com.lightbend.paradox.markdown.{LeafBlockDirective, _}` below